test/puppetlabs/puppetdb/http/reports_test.clj

```diff
@@ -388,6 +388,46 @@
                                 (assoc basic2 :hash hash2)])
      munge-reports-for-comparison)))
 
+(deftestseq latest-report-queries
+  [[version endpoint] endpoints]
+  (let [basic (:basic reports)
+        hash1 (:hash (store-example-report! basic (now)))
+        basic2 (:basic2 reports)
+        hash2 (:hash (store-example-report! basic2 (now)))
+        latest (get-response endpoint ["=" "latest_report?" true])
+        latest-body (json/parse-strict-string (:body latest) true)
+        latest2 (get-response endpoint ["and" ["=" "latest_report?" true] ["=" "noop" false]])
```

Comment:
not through this mechanism. I updated the release notes and the docs to be a bit more explicit like we discussed.